### PR TITLE
Docs 1764

### PR DESCRIPTION
**Pull request by Kris-MultiSafepay**
### Basic
- We only accept documentation that is proved to work on our LIVE API
- At least 1 approval is needed before pull requests can be merged
- The article must match our Synonym word list. For external commits, a MultiSafepay employee will check
- US English must be applied
- Use bullets for numerations where the sort order doesn't matter. For chronological order, use numbers. This does not apply to changelogs
- Set link on email address.

### Text style
- Paths are written like: _Configuration → System → Plugin → MultiSafepay_
- Use single quotation marks in your Markdown code (example title: 'Title of the article')
- Use Italics for buttons, clicks, paths, etc. Do mind to close the italics before the end of a sentence followed by a period
- Numerations / bullets are punctuated with a period at the end of the numerations / bullets.
- Periods should be excluded for sentences that finish in an e-mail address (e.g. "..contact us at <example@example.com> "  )

### Images
- Screenshot recomended sizes: Width: 820px - Height: auto

### Aliases
- When renaming or deleting a page, add an alias of the renamed or deleted page to the page users should be redirected to instead. This prevents external links to return 404 errors. Check the [Hugo Documentation](https://gohugo.io/content-management/urls/#aliases) for more information.

### CODE REVIEW

File: content/tools/transactions-api-endpoint/_index.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+| debit_credit | `D` |
+| type (transaction type) | `refund` |
+
+Use parameters `end` and `start` to specify a range of dates.
+
+{{< br >}}
+
+## Support
+
+If you have any issues or questions, [contact our integration team](mailto:integration@multisafepay.com)
```
Comment (Kris-MultiSafepay) [inline content/tools/transactions-api-endpoint/_index.md:161]:
```suggestion
If you have any issues or questions, [contact our Integration team](mailto:integration@multisafepay.com)
```

File: content/tools/transactions-api-endpoint/_index.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+| debit_credit | `D` |
+| type (transaction type) | `refund` |
+
+Use parameters `end` and `start` to specify a range of dates.
+
+{{< br >}}
+
+## Support
+
+If you have any issues or questions, [contact our Integration team](mailto:integration@multisafepay.com)
```
Comment (Kris-MultiSafepay) [inline content/tools/transactions-api-endpoint/_index.md:161]:
```suggestion
If you have any issues or questions, [contact our Integration Team](mailto:integration@multisafepay.com)
```

File: content/tools/transactions-api-endpoint/_index.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
+---
+title : "Transactions API endpoint"
+layout : "single"
+tags : "hidden"
+---
+
+## Introduction
+
+The transactions API endpoint returns the data of transactions under your account. Using this endpoint, you can automate business operations by matching records at a transaction level. This page provides information about the endpoint and its usage.
```
Comment (Atiq-MultiSafepay) [inline content/tools/transactions-api-endpoint/_index.md:9]: The transactions API endpoint returns the data of transactions that are associated with your account. By using this endpoint, you can automate business operations by matching records at a transaction level. This page provides information about the endpoint and its usage.

**Review by Atiq-MultiSafepay — CHANGES_REQUESTED** [review]

File: content/tools/transactions-api-endpoint/_index.md (hunk 3 above)
Comment (Kris-MultiSafepay) [inline content/tools/transactions-api-endpoint/_index.md:9]:
```suggestion
The transactions API endpoint returns the data of transactions under your account. Using this endpoint, you can automate business operations by matching records at a transaction level. This page provides information about the endpoint and its usage.
```

**Pull request merged**